### DISCUSSION TITLE
Encoding A2DP stream to SBC and containarizing in Binaryencoder container and sending it via serial and decoding it back in another ESP32 and sending the decoded output to A2DP source?

### DISCUSSION THREAD

**Discussion by stealth86** [pschatzmann/arduino-audio-tools#1700] [Q&A] [answered]
The first ESP32 is encoding and sending the data via serial. The second ESP32 is receiving it and decoding, but the A2DPStream is not playing any audio. What would be the pipeline required for the above req?

**Comment by stealth86** [comment, marked as answer]
[https://github.com/stealth86/esp32-BT/tree/master/GazeboLights](url) - input A2DPstream code
[https://github.com/stealth86/esp32-BT/tree/master/streams_serial_a2dp](url) - Output A2DP code

**Comment by pschatzmann** [comment]
I suggest to perform a proper [error analysis and build your sketch step by step.](https://github.com/pschatzmann/arduino-audio-tools/wiki/It's-not-working)
